src/glfw3.jl

```diff
@@ -437,6 +437,7 @@ const _window_callbacks = Dict{Window, Ref{Vector{Callback}}}()
 function CreateWindow(width::Integer, height::Integer, title::AbstractString, monitor::Monitor=Monitor(C_NULL), share::Window=Window(C_NULL))
 	window = ccall( (:glfwCreateWindow, lib), Window, (Cint, Cint, Cstring, Monitor, Window), width, height, title, monitor, share)
 	callbacks = Ref{Vector{Callback}}(fill(nothing, _window_callback_num[]))
+    MakeContextCurrent(window)
```

Comment:
It looks like this might be indented with spaces instead of tabs.

Reply:
ah probably shouldn't be in there anyways!